packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx

```diff
@@ -22,15 +31,31 @@ interface Props extends ThemeProps {
 function SaveMnemonic({ className, onNextStep, onPreviousStep, seed }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const { show } = useToast();
+
+  const [isSecretCopied, setIsSecretCopied] = useState(false);
+
   const seedArray = seed.split(' ');
 
-  const _onCopy = useCallback((test: string, success: boolean) => {
-    if (success) {
-      show(t('Secret phrase copied to clipboard'), 'success');
-    } else {
-      show(t('Failed copying to clipboard'), 'critical');
-    }
-  }, [show, t]);
+  const _onCopy = useCallback(
+    (test: string, success: boolean) => {
+      if (success) {
+        show(t('Secret phrase copied to clipboard'), 'success');
```

Comment:
How about going crazy and:
```suggestion
  const _onCopy = useCallback(
    (test: string, success: boolean) => {
      if (success) {
        setIsSecretCopied(true)
        show(t('Secret phrase copied to clipboard'), 'success');
```
🤖

Reply:
I understood this checkbox differently, more like "As a user I've made sure that the secret is saved and secure".
Coping to clipboard isn't enough for me, wdyt?

Reply:
Fair enough 👍